src/controllers/alphabetical-search/search.controller.ts

```diff
@@ -79,6 +81,15 @@ const route = async (req: Request, res: Response) => {
                     }
                 ];
             });
+
+            if (req.query.originalCompanyNumber) {
+                originalCompanyNumber = req.query.originalCompanyNumber;
+            };
```

Comment:
is this semi colon on line 87 needed? I didn't think it was for if statement brackets?